#! /bin/sh /usr/share/dpatch/dpatch-run
## 14_create-pgsql-indices.dbscripts.dpatch
#
# Description: Uncomments the creation of two indexes in the pgsql database
#  creation scripts, since the PostgreSQL builds are geared towards large
#  setups. Until v3.0.3, the script suggested a third index
#  on (jobid, pathid, filenameid).
# Origin: vendor, file:///dev/null
# Author: John Goerzen <jgoerzen@complete.org>
# Author: Lucas B. Cohen <lbc@members.fsf.org>
# Reviewed-by: John Goerzen <jgoerzen@complete.org>
# Reviewed-by: Lucas B. Cohen <lbc@members.fsf.org>
# Last-Update: 2009-12-03
#
# 2009.12.7 : tweaked patch to a patchlevel 2, to accomodate dpatch and git
#
## DP: generated from bacula 3.0.2-3 diff.gz with dpatch-convert-diffgz 2.0.31

@DPATCH@

--- a/bacula/src/cats/make_postgresql_tables.in
+++ b/bacula/src/cats/make_postgresql_tables.in
@@ -52,9 +52,9 @@
 -- Possibly add one or more of the following indexes
 --  if your Verifies are too slow.
 --
--- CREATE INDEX file_pathid_idx on file(pathid);
--- CREATE INDEX file_filenameid_idx on file(filenameid);
+CREATE INDEX file_pathid_idx on file(pathid);
+CREATE INDEX file_filenameid_idx on file(filenameid);

 CREATE TABLE job
 (